<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1346</width>
    <height>878</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>POS</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="master_data_button">
         <property name="text">
          <string>Master Data</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="settings_button">
         <property name="text">
          <string>Settings</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="transaction_button">
         <property name="text">
          <string>Transaction</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="report_button">
         <property name="text">
          <string>Report</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="log_button">
         <property name="text">
          <string>Log</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="import_button">
         <property name="text">
          <string>Import</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="export_button">
         <property name="text">
          <string>Export</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="backup_button">
         <property name="text">
          <string>Backup</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>448</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_5">
         <property name="text">
          <string>Logout</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="master_data_page">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QWidget" name="widget_4" native="true">
             <property name="styleSheet">
              <string notr="true">QWidget{
	background-color:grey;
}</string>
             </property>
             <widget class="QLabel" name="label_2">
              <property name="geometry">
               <rect>
                <x>500</x>
                <y>330</y>
                <width>291</width>
                <height>81</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">QLabel{
	font-size:24px;
	font-weight:bold;
	color:white;
}</string>
              </property>
              <property name="text">
               <string>MASTER DATA</string>
              </property>
             </widget>
             <widget class="QPushButton" name="products_button">
              <property name="geometry">
               <rect>
                <x>50</x>
                <y>50</y>
                <width>191</width>
                <height>61</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
	color: white;
	font-size:24px;
}</string>
              </property>
              <property name="text">
               <string>Products</string>
              </property>
             </widget>
             <widget class="QPushButton" name="categories_button">
              <property name="geometry">
               <rect>
                <x>280</x>
                <y>50</y>
                <width>191</width>
                <height>61</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
	color: white;
	font-size:24px;
}</string>
              </property>
              <property name="text">
               <string>Categories</string>
              </property>
             </widget>
             <widget class="QPushButton" name="suppliers_button">
              <property name="geometry">
               <rect>
                <x>530</x>
                <y>50</y>
                <width>191</width>
                <height>61</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
	color: white;
	font-size:24px;
}</string>
              </property>
              <property name="text">
               <string>Suppliers</string>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="transaction_page">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QWidget" name="widget_5" native="true">
          <property name="styleSheet">
           <string notr="true">QWidget{
	background-color:grey;
}</string>
          </property>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>260</x>
             <y>190</y>
             <width>291</width>
             <height>81</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel{
	font-size:24px;
	font-weight:bold;
	color:white;
}</string>
           </property>
           <property name="text">
            <string>Transaction</string>
           </property>
          </widget>
          <widget class="QPushButton" name="transactions_button">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>90</y>
             <width>191</width>
             <height>61</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	color: white;
	font-size:24px;
}</string>
           </property>
           <property name="text">
            <string>Transactions</string>
           </property>
          </widget>
          <widget class="QPushButton" name="transactions_list_button">
           <property name="geometry">
            <rect>
             <x>290</x>
             <y>90</y>
             <width>191</width>
             <height>61</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	color: white;
	font-size:24px;
}</string>
           </property>
           <property name="text">
            <string>Transactions List</string>
           </property>
          </widget>
          <widget class="QPushButton" name="purchasing_button">
           <property name="geometry">
            <rect>
             <x>530</x>
             <y>90</y>
             <width>191</width>
             <height>61</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	color: white;
	font-size:24px;
}</string>
           </property>
           <property name="text">
            <string>Purchasing</string>
           </property>
          </widget>
          <widget class="QPushButton" name="purchasing_list_button">
           <property name="geometry">
            <rect>
             <x>760</x>
             <y>90</y>
             <width>191</width>
             <height>61</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	color: white;
	font-size:24px;
}</string>
           </property>
           <property name="text">
            <string>Purchasing List</string>
           </property>
          </widget>
          <widget class="QPushButton" name="roles_button">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>280</y>
             <width>191</width>
             <height>61</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	color: white;
	font-size:24px;
}</string>
           </property>
           <property name="text">
            <string>Roles</string>
           </property>
          </widget>
          <widget class="QPushButton" name="role_permissions_button">
           <property name="geometry">
            <rect>
             <x>300</x>
             <y>280</y>
             <width>191</width>
             <height>61</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	color: white;
	font-size:24px;
}</string>
           </property>
           <property name="text">
            <string>Role Permissions</string>
           </property>
          </widget>
          <widget class="QPushButton" name="customers_button">
           <property name="geometry">
            <rect>
             <x>540</x>
             <y>280</y>
             <width>191</width>
             <height>61</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	color: white;
	font-size:24px;
}</string>
           </property>
           <property name="text">
            <string>Customers</string>
           </property>
          </widget>
          <widget class="QPushButton" name="logs_button">
           <property name="geometry">
            <rect>
             <x>770</x>
             <y>280</y>
             <width>191</width>
             <height>61</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	color: white;
	font-size:24px;
}</string>
           </property>
           <property name="text">
            <string>Logs</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="settings_page">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QWidget" name="widget_3" native="true">
          <property name="styleSheet">
           <string notr="true">QWidget{
	background-color:grey;
}</string>
          </property>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>260</x>
             <y>190</y>
             <width>291</width>
             <height>81</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel{
	font-size:24px;
	font-weight:bold;
	color:white;
}</string>
           </property>
           <property name="text">
            <string>SETTINGS</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1346</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
